Python/Product/Analysis/Parsing/Ast/MemberExpression.cs

```diff
@@ -76,7 +76,11 @@ public sealed class MemberExpression : Expression {
             res.Append('.');
             if (!this.IsIncompleteNode(ast)) {
                 res.Append(this.GetSecondWhiteSpaceDefaultNull(ast));
-                res.Append(this.GetVerbatimImage(ast) ?? _name);
+                if (format.UseVerbatimImage) {
+                    res.Append(this.GetVerbatimImage(ast) ?? _name);
```

Comment:
We don't actually want to round-trip here. What's needed is to get the actual expression with mangled identifiers (as a string, to send to evaluator), given a span in the original source.